OWML.ModHelper.Menus/ModInputCombinationPopup.cs

```diff
@@ -0,0 +1,201 @@
+﻿using OWML.Common;
+using OWML.Common.Menus;
+using OWML.ModHelper.Input;
+using OWML.ModHelper.Events;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System;
+using System.Collections.ObjectModel;
+using System.Reflection;
+
+namespace OWML.ModHelper.Menus
+{
+    public class ModInputCombinationPopup : PopupMenu
+    {
+        public ILayoutManager Layout { get; private set; }
+        public event Action OnPopupReset;
+        public string Combination
+        {
+            get
+            {
+                string result = "";
+                for (var j = 0; j < _combination.Count; j++)
+                {
+                    result += ModInputLibrary.KeyCodeToString(_combination[j]);
+                    if (j < _combination.Count - 1)
+                    {
+                        result += "+";
+                    }
+                }
+                return result;
+            }
+        }
```

Comment:
`public string Combination => string.Join("+", _combination.Select(ModInputLibrary.KeyCodeToString));`

Reply:
Much better that way

Reply:
Did the same in CombinationMenu